src/config/config.json

```diff
@@ -0,0 +1,30 @@
+{
+    "development": {
+      "username": "postgres",
+      "password": "admin",
+      "database": "testdb",
+      "host": "localhost",
+      "dialect": "postgres",
+      "operatorsAliases": false
+    },
+    "test": {
+      "username": "root",
+      "password": null,
+      "database": "database_test",
+      "host": "127.0.0.1",
+      "dialect": "mysql",
+      "operatorsAliases": false
+    },
+    "production": {
```

Comment:
Consider separating your dev and prod configs.
E.g. config.common.json, congif.dev.json and config.prod.json.
Currently, you have a major security issues since your id, username and password for EC2 instance are publicly available.

Reply:
resolved